python/lsst/obs/hsc/hscMapper.py

```diff
@@ -148,12 +149,12 @@ def map(self, datasetType, dataId, write=False):
         return location
 
     @staticmethod
-    def _flipChipsLR(exp, wcs, dataId, dims=None):
+    def _flipChipsLR(exp, wcs, detectorId, dims=None):
```

Comment:
Please add parameters/returns to docstring

Reply:
I'd rather not try to guess at gen2 butler method definitions. They're also going away soon (I so very much hope!). My same comment applies to the other pre-existing methods in every `CameraMapper` ever...